### ISSUE TITLE
No unhandled errors error thrown when using `takeLeading`

### ISSUE THREAD

**Issue by ilias-t** [pke/eslint-plugin-redux-saga#64]
Version: v1.1.1
Related issues: #40 

Please let me know if I'm using `takeLeading` incorrectly, but I don't believe I am. With the following code, an error is thrown with the message:
>A Saga must handle its effects' errors (use try/catch)eslint(redux-saga/no-unhandled-errors)

```javascript
function* rootSaga() {
  try {
    yield all([
      takeEvery(Action.foo, saga1),
      // Only this line throws
      takeLeading(Action.bar, saga2),
      takeLatest(Action.baz, saga3),
    ]);
  } catch (e) {
    // ...
  }
```

**Comment by pke** [comment]
What you mean the `takeLeading` line throws? I have run this in a test and it does not trigger the rule. So when you remove the `takeLeading` line it does not trigger?

**Comment by ilias-t** [comment]
Yes, eslint is failing only for `takeLeading`—neither `takeEvery` or `takeLatest` has the same issue.

Error message:
![image](https://user-images.githubusercontent.com/4505008/70364765-c3cb2500-1842-11ea-83ef-226f54fb7559.png)

The only way to get around this seems to be forcefully `// eslint-disable-next-line redux-saga/no-unhandled-errors` every time `takeLeading` is used.

**Comment by pke** [comment]
Somehow I can't reproduce this in the plugins tests. Guess I'll have to try again.

**Comment by pke** [comment]
Fixed with #63. Releasing new version. Please check and confirm here @ilias-t

**Comment by ilias-t** [comment]
Looking at the diff, it's likely fixed. The project I'm seeing this on is pointing to an internal registry that's refreshed only a few times a day so I'll test this again tomorrow and close if I can't repro. Thanks!

**Comment by ilias-t** [comment]
v1.1.3 fixes the issue—thanks again!
